From 7b177ffdc4ffa000b4697be7404e03df70a6c8e4 Mon Sep 17 00:00:00 2001
From: David Wagner <david.wagner@intel.com>
Date: Mon, 10 Feb 2014 12:23:27 +0100
Subject: Activate the "XInclude" feature by default

BZ: 168727

This is the default value in the vanilia libxml2 project.

Change-Id: I09b7d330336446c2419e2ef272dd7fa736e6c8f2
Category: device enablement
Domain: AudioComms-Common
Origin: internal
Upstream-Candidate: yes
Signed-off-by: David Wagner <david.wagner@intel.com>
---
 include/libxml/xmlversion.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/include/libxml/xmlversion.h b/include/libxml/xmlversion.h
index 9e6f66b..10ed269 100644
--- a/include/libxml/xmlversion.h
+++ b/include/libxml/xmlversion.h
@@ -255,7 +255,7 @@ XMLPUBFUN void XMLCALL xmlCheckVersion(int version);
  *
  * Whether XInclude is configured in
  */
-#if 0
+#if 1
 #define LIBXML_XINCLUDE_ENABLED
 #endif
 
-- 
1.7.4.1

